app/views/new_document/choose_document_type.html.erb

```diff
@@ -3,7 +3,7 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= form_tag create_document_path do %>
+    <%= form_tag create_document_path, data: { gtm: "new_document" } do %>
```

Comment:
minor nitpick, but we've gone with kebab case on all the previous gtm attributes

Reply:
Fixed in 63c57810ee3b5f7e31cafb73611c5e5ea256067f